SimpleExec/ProcessStartInfo.cs

```diff
@@ -25,5 +28,10 @@ internal static class ProcessStartInfo
                     RedirectStandardError = false,
                     RedirectStandardOutput = captureOutput
                 };
+
+            configureEnvironment?.Invoke(startInfo.Environment);

```

Comment:
There is a nicer `startInfo.EnvironmentVariables` property but it's not available on netstandard1.3

Reply:
SimpleExec targets `netstandard2.0`, which has `ProcessStartInfo.EnvironmentVariables`. But are you sure it's nicer? `EnvironmentVariables` is a `StringDictionary` which seems to be a backwards-compatibility hangover from the pre-generic days. It implements only `IEnumerable` and not `IEnumerable<string>`. 🤢